packages/mds-audit/api.ts

```diff
@@ -191,52 +187,52 @@ function api(app: express.Express): express.Express {
           } = req.body
 
           // Validate input params
-          if (
-            isValidTimestamp(timestamp) &&
-            isValidProviderId(provider_id) &&
-            isValidProviderVehicleId(provider_vehicle_id) &&
-            isValidAuditEventId(audit_event_id) &&
-            isValidAuditDeviceId(audit_device_id) &&
-            isValidTelemetry(telemetry, { required: false })
-          ) {
-            // Find provider device and event by vehicle id lookup
-            const provider_device = await getVehicle(provider_id, provider_vehicle_id)
-            const provider_device_id = provider_device ? provider_device.device_id : null
-            const provider_name = providerName(provider_id)
+          isAuditApiAuditStartRequestValid({
```

Comment:
Does this throw an error if it's invalid?

Reply:
yeah! changed these over to the form that asserts/throws. Thanks!